cli/src/commands/auth.rs

```diff
@@ -216,7 +216,7 @@ pub async fn handle_auth_create_token(
 
     // Try to parse expiry.
     let expiry = match expiry.map(|expiry| expiry.parse()) {
-        Some(Ok(expiry)) => Some(Utc::now() + Duration::days(expiry)),
+        Some(Ok(expiry)) => Some(Utc::now() + Duration::try_days(expiry).unwrap()),
```

Comment:
I think chrono marked these as deprecated to avoid panics.

```rust
    // Try to parse expiry.
    let expiry = expiry
        .map(|expiry| {
            Utc::now()
                .checked_add_days(Days::new(expiry.parse()?))
                .ok_or_else(|| anyhow!("expiration is too far into the future"))
        })
        .transpose();
    let expiry = match expiry {
        Ok(expiry) => expiry,
        Err(err) => {
            print_user_failure!("invalid token expiry: {err}");
            return Ok(ExitCode::InvalidTokenExpiration);
        },
    };
```

Reply:
It has been changed to this suggestion. Thank you for providing it.